### DISCUSSION TITLE
Vercel Analytics "Awaiting Data"

### DISCUSSION THREAD

**Discussion by martinbauwens** [vercel/community#3992] [Help]
### Summary

Hello,

I enabled Vercel Analytics. 

However, it remains in  "Awaiting Data" mode with no prospect of moving forward.

<img width="587" alt="Screenshot 2023-09-09 at 19 34 27" src="https://github.com/vercel/community/assets/143646690/cbf56bf7-227a-4050-9ad4-41a596914156">

Regards,

Martin

### Steps to Reproduce

See above.

**Comment by amyegan** [comment]
Hey @martinbauwens. Is this still happening?

**Comment by martinbauwens** [comment]
Yes, it is still happening.

**Reply by amyegan** [reply]
Are you able to share the site address here?
